spec/services/service_offering/add_to_portfolio_item_spec.rb

```diff
@@ -23,7 +23,7 @@
     item = add_to_portfolio_item.process.item
 
     expect(item).to be_a(PortfolioItem)
-    expect(item.attributes.count).to eql(17)
+    expect(item.attributes.count).to eql(18)
```

Comment:
@lindgrenj6 - I think we can remove this spec.  I know it's been around for a while but it isn't testing any real behavior in my opinion.

Reply:
Removing this!